scripts/sanitycheck

```diff
@@ -802,6 +802,16 @@ class DeviceHandler(Handler):
         except subprocess.CalledProcessError:
             os.write(write_pipe, b'x')  # halt the thread
 
+        if post_script:
+            with subprocess.Popen(post_script, stderr=subprocess.PIPE, stdout=subprocess.PIPE) as proc:
+                try:
+                    (stdout, stderr) = proc.communicate(timeout=30)
```

Comment:
Do

```python
stdout, _ = proc.communicate(timeout=30)
```

`stderr` is unused here. It's just shadowed by it existing as a local variable. `pylint` knows not to flag `_` as unused.

A nice side effect of breaking some things out into separate functions would be to catch stuff like that easier.